ivy_tests/test_ivy/test_functional/test_core/test_dtype.py

```diff
@@ -103,10 +157,62 @@ def test_broadcast_to(
         fw,
         "broadcast_to",
         x=x,
-        shape=array_shape,
+        shape=to_shape,
     )
 
 
+# # broadcast_to - attempt to have only one data.draw in function
```

Comment:
We should remove this commented-out code when merging I guess? It's replaced by another implementation it seems.